prqlc/prqlc-parser/Cargo.toml

```
@@ -12,10 +12,13 @@ version.workspace = true
 doctest = false
 
 [dependencies]
-itertools = {workspace = true}
-prqlc-ast = {path = "../prqlc-ast", version = "0.11.5" }
-semver = {version = "1.0.22"}
+prqlc-ast = {path = "../prqlc-ast"}
```

```suggestion
prqlc-ast = {path = "../prqlc-ast", version = "0.11.5" }
```
otherwise we can't publish

---

FYI @m-span we missed this and it broke the publishing! (No great stress tbc, wanted to let you know out of respect rather than blame...)

---

🫨 whoops!